src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +125,15 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts list by alphabetical order of names.
+     * @param NameComparator
+     */
+
+    public void sort(Comparator<Person> NameComparator) {
+        Collections.sort(internalList, NameComparator);
```

Comment:
You can have the (inline) Class implementation of NameComparator here which is **only** used by this method rather declaring it in SortNameCommand and passing as an argument.